### PR TITLE
PR regarding Vagrantfile

### PR DESCRIPTION
**Pull request by ronityosefia**
I did the Vagrantfile step, please approve so we can continue to our next step

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,3 +130,10 @@ dmypy.json
 
 # Vagrant
 .vagrant/
+<<<<<<< HEAD
```
Comment (ravidbro) [inline .gitignore:133]: This isn't code.
This is git way to tell you that you have a conflict that needs to be resolved

Comment (omernu) [reply .gitignore:133]: hi, we've already fixed the conflicts. is there any new ones?

Comment (ravidbro) [reply .gitignore:133]: The '<<<< HEAD' and '>>>>>' were created by git to show you the area that needs to be resolved,
When you resolve it, you need to remove these lines.
Usually (unless it's a text file), the code won't work with these lines inside.

**Comment by simzacks** [comment]
This was closed because of PR #15

**Pull request closed without merging**